sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### New Features
 - Added `DatasourceCredential` CRUD operations to the `MetricsAdvisorAdministrationClient`. This API provides new ways of authenticating to a `DataFeedSource`.
+- Added property `Authentication` to data sources `AzureBlobDataFeedSource`, `AzureDataExplorerDataFeedSource`, `AzureDataLakeStorageGen2DataFeedSource`, and `SqlServerDataFeedSource` to specify the authentication type to use.
```

Comment:
are you thinking of adding some samples to show this and guide the user on the most common data sources and the different authentication types?

Reply:
Yes. Already tracked by https://github.com/Azure/azure-sdk-for-net/issues/21586.